application/schemas/templates/components.py

```diff
@@ -28,8 +41,89 @@ class Component(BaseModel):
                     'a higher priority.',
     )
 
-    class Config:
```

Comment:
You have lost setting that forbids additional parameters in Helm and HTTP components.